Sources/Clickstream/Core/Interface/ClickStream.swift

```diff
@@ -36,7 +36,6 @@ public final class Clickstream {
     /// Clickstream shared instance.
     private static var sharedInstance: Clickstream?
     
-    #if EVENT_VISUALIZER_ENABLED
```

Comment:
Why we have removed this macro?

Reply:
I had done it for testing, it is now fixed